best-practices.md

```diff
@@ -0,0 +1,21 @@
+# Best Practices
+
+## [CISL Best Practices](https://www2.cisl.ucar.edu/user-support/cisl-best-practices)
+
+If you are using NCAR Computational and Information Systems Lab (CISL) resources such as Casper/Cheyenne, this is a great place to start with learning more making the most of your computing and storage allocations.
+
+## Helpful Templates
+
+Thinking about starting a new project? It can be helpful to use a template so you are not starting from scratch.
+
+### [Package Template](https://github.com/andersy005/template)
+
+Anderson Banihirwe created a template repository for packages, which includes automated code-quality checkers. This can be a helpful place to start.
+
+### [Analysis Repository](https://github.com/jbusecke/cookiecutter-science-project)
+
+Within the Pangeo community, Julius Busecke created the "Cookiecutter-Science-Project" repository which serves as a template when you are working with Jupyter notebooks. This repository walks you through setting up your repository and syncing it up through Github.
+
+## [Software Design](https://docs.google.com/presentation/d/1hVCuxtm0f8TzDqsy2kC5nnjqNt3rsz26qgcMQOnM0RY/edit?usp=sharing)
```

Comment:
I think Software Design should be the `##` header, and then this link should be in `### 4+1 View Model`. Maybe something like

```
## Software Design

### [4+1 View Model](https://docs.google.com/presentation/d/1hVCuxtm0f8TzDqsy2kC5nnjqNt3rsz26qgcMQOnM0RY/edit?usp=sharing)

Currently, the ESDS group is experimenting with design documents for the next generation of diagnostics package. The linked set of slides is an example of the 4+1 View Model of Software Architecture with an application to the diagnostics workflow. Feel free to use this example as a template!
```

This will allow us to add other design documents to the page as they are created.

Reply:
Just fixed that - added in this other scientific software development link